src/ui/Base/Component.ts

```diff
@@ -454,7 +454,14 @@ export class Component extends BaseComponent {
   static resolveRoot(element: HTMLElement): HTMLElement {
     Assert.exists(element);
     const resolvedSearchInterface = Component.resolveBinding(element, SearchInterface);
```

Comment:
Technically, we could omit using `Component.resolveBinding`, I just kept it to prevent any breaking behaviour.